examples/mplot3d/lorenz_attractor.py

```diff
@@ -18,45 +18,41 @@
 import matplotlib.pyplot as plt
 
 
-def lorenz(x, y, z, s=10, r=28, b=2.667):
+def lorenz(xyz, s=10, r=28, b=2.667):
```

Comment:
Maybe keyword-only so you don't accidentally `*` in like below.
```suggestion
def lorenz(xyz, *, s=10, r=28, b=2.667):
```

Reply:
I think this is still unresolved...

Reply:
Thanks, I had missed that.